src/views/createCommunity/index.tsx

```diff
@@ -1158,7 +1157,7 @@ function CreateCommunityScreen() {
                                     }}
                                     source={require('../../assets/images/waitingTx.gif')}
                                 />
-                            ) : sendingSuccess ? (
+                            ) : !sendingSuccess ? (
```

Comment:
is this correct?

Reply:
not, it's not. Just fixed. That was a test and I forgot to revert it back.